public/client.js

```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
     }
   });
   
+  /* REMOVE MESSAGE BEGIN */
+  const chat = document.querySelector('.chat');
+  console.log(messages.data.length);
+  if(chat && messages.data.length < 25) {
+    chat.innerHTML += `<div class="message flex flex-row">
+      <img src="https://adaptable.io/img/color logo.svg" alt="Adaptable.io logo" class="avatar">
+      <div class="message-wrapper">
+      <p class="message-header">
```

Comment:
Indentation seems off here and with the `<ul>` tag. Maybe elsewhere too.

Reply:
Does this indentation look better.  Man I cannot believe how dependent I am on an autoformatter :)

Reply:
Fixed line 245 as well.